tensorflow/lite/experimental/micro/examples/hello_world/Makefile.inc

```diff
@@ -1,4 +1,4 @@
-HELLO_WORLD_TEST_SRCS := \
+HELLO_WORLD_TEST_SRCS += \
```

Comment:
I think these should stay the same - please see the comment on the root platform riscv makefile. Sorry we don't have a great canonical example - but this PR can be!

Reply:
Please revert these changes for all of the root example `Makefile.inc`

Reply:
I have revert these changes in the new patch version. The branch is here https://github.com/zhoupeng/tensorflow/tree/fix_wrapfunction .
Sorry, I don't if I need to create new PR after update the patch as you review suggestion.